caffe_app/views/import_prototxt.py

```diff
@@ -469,6 +469,16 @@ def ContrastiveLoss(layer):
     return params
 
 
+def Concat(layer):
+    params = {}
+    if (layer.concat_param.axis is not None):
+        params['axis'] = layer.concat_param.axis
+    else:
+        # default value for axis of dropout in caffe
```

Comment:
How is this related to dropout?

Reply:
Sorry I accidentally wrote dropout there it should be concat